spec/jobs/tl3_promotions_spec.rb

```diff
@@ -24,8 +24,11 @@ def create_qualifying_stats(user)
     run_job
   end
 
-  it "promotes a qualifying tl2 user who has a group_locked_trust_level" do
-    _group_locked_user = Fabricate(:user, trust_level: TrustLevel[2], group_locked_trust_level: TrustLevel[1])
+  it "promotes a qualifying tl2 user who has a group_granted_trust_level" do
+    group = Fabricate(:group, grant_trust_level: 1)
+    _group_locked_user = Fabricate(:user, trust_level: TrustLevel[2])
+    group.add(_group_locked_user)
```

Comment:
I think we don't need to use an undescore before the variable name here since we're using the variable.